spec/system/school/courses/target_version_management_spec.rb

```diff
@@ -13,250 +11,109 @@
   let!(:school_admin) { create :school_admin, school: school }
   let!(:level_1) { create :level, :one, course: course }
   let!(:target_group_1) { create :target_group, level: level_1 }
-  let!(:target_1) { create :target, target_group: target_group_1 }
-  let!(:target_2) { create :target, target_group: target_group_1 }
-  let(:sample_markdown_text) { Faker::Markdown.sandwich(6) }
+  let!(:target_1) { create :target, :with_content, target_group: target_group_1 }
+  let!(:coach) { create :faculty, school: school }
+  let!(:course_author) { create :course_author, course: course, user: coach.user }
 
-  # Create content blocks for target_1 for latest version
-  let!(:cb_1) { create :content_block, :image }
-  let!(:cb_2) { create :content_block, :markdown }
-  let!(:cb_3) { create :content_block, :file }
-  let!(:cb_4) { create :content_block, :embed }
-
-  # Create few content blocks for target_1 for old versions
-  let!(:cb_5) { create :content_block, :file, created_at: 3.days.ago }
-  let!(:cb_6) { create :content_block, :embed, created_at: 3.days.ago }
-
-  let!(:cb_7) { create :content_block, :file, created_at: 2.days.ago }
-
-  before do
-    stub_request(:get, 'https://www.youtube.com/oembed?format=json&url=https://www.youtube.com/watch?v=3QDYbQIS8cQ').to_return(body: '{"version":"1.0","provider_name":"YouTube","html":"\u003ciframe width=\"480\" height=\"270\" src=\"https:\/\/www.youtube.com\/embed\/3QDYbQIS8cQ?feature=oembed\" frameborder=\"0\" allow=\"accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture\" allowfullscreen\u003e\u003c\/iframe\u003e","thumbnail_url":"https:\/\/i.ytimg.com\/vi\/3QDYbQIS8cQ\/hqdefault.jpg","provider_url":"https:\/\/www.youtube.com\/","thumbnail_height":360,"type":"video","height":270,"thumbnail_width":480,"author_url":"https:\/\/www.youtube.com\/channel\/UCvsvW3QH1700y-j2VfEnq-A","author_name":"Just smile","title":"Funny And Cute Cats - Funniest Cats Compilation 2019","width":480}', status: 200)
-  end
-
-  def file_path(filename)
-    File.absolute_path(Rails.root.join('spec', 'support', 'uploads', 'files', filename))
-  end
-
-  def latest_content_versions(target)
-    target.content_versions.reload.where(version_on: target.latest_content_version_date)
-  end
+  # rubocop:disable Rails/SkipsModelValidations
+  context 'admin visits target version page' do
+    scenario 'creates a target version', js: true do
```

Comment:
Remove `js: true` option from all the scenarios in this spec; the main `feature` already has this option set.